lib/peeps.rb

```diff
@@ -0,0 +1,14 @@
+require 'pg'
+class Peep
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
```

Comment:
Consider creating a Database Connector to remove this from your methods (step 14 Bookmark Manager challenge)